### PR TITLE
refactor: Clean after code review

### PR DESCRIPTION
**Pull request by serereg**
Clean after code review.

Relates to #66
Relates to #170

### CODE REVIEW

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py [hunk 1]
```diff
@@ -85,18 +85,18 @@ class Prediction(BaseModel):
...
+    id: int = 1
     completed_by: int = 1
     result: List[ResultItem]
     was_cancelled: bool = False
     ground_truth: bool = False
-    created_at: str = "2022-12-13T09:57:08.451845Z"
-    updated_at: str = "2022-12-13T09:57:08.451875Z"
-    lead_time: float = 121.434
+    created_at: str = ""
+    updated_at: str = ""
```
Comment (spiridonovfed) [inline convert/src/label_studio_to_badgerdoc/models/label_studio_models.py:94]: According to this these values should not be hardcoded with default values:
https://github.com/epam/badgerdoc/pull/117#discussion_r1062214735

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py [hunk 2]
```diff
@@ -85,18 +85,18 @@ class Prediction(BaseModel):
 
 
 class Annotation(BaseModel):
-    id: int = 7
+    id: int = 1
     completed_by: int = 1
```
Comment (spiridonovfed) [inline convert/src/label_studio_to_badgerdoc/models/label_studio_models.py:89]: These values should not be hardcoded with default values according to this:
https://github.com/epam/badgerdoc/pull/117#discussion_r1062214438

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py [hunk 3]
```diff
@@ -124,8 +124,8 @@ class ModelItem(BaseModel):
     predictions: List[int] = []
     data: Data
     meta: Meta = Meta()
-    created_at: Optional[str] = "2022-12-13T09:57:08.451845Z"
-    updated_at: Optional[str] = "2022-12-13T09:57:08.451845Z"
+    created_at: Optional[str] = ""
+    updated_at: Optional[str] = ""
     inner_id: int = 1
     total_annotations: int = 0
     cancelled_annotations: int = 0
```
Comment (spiridonovfed) [inline convert/src/label_studio_to_badgerdoc/models/label_studio_models.py:131]: This and all attributes with default values should be rechecked:
https://github.com/epam/badgerdoc/pull/117#discussion_r1061264133

File: convert/tests/test_label_studio/test_import_text.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    response = test_app.post(
+        "/text/import",
+        json=(test_request_payload),
+    )
+    assert response.status_code == 201
+
+
+def test_import_empty_text():
+    badgerdoc_format = BadgerdocFormat()
+    badgerdoc_format.convert_from_text("")
```
Comment (spiridonovfed) [inline convert/tests/test_label_studio/test_import_text.py:42]: Please see and work all the comments from https://github.com/epam/badgerdoc/issues/170

**Review by spiridonovfed — CHANGES_REQUESTED** [review]

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py [hunk 5]
```diff
@@ -20,7 +20,7 @@ class LabelStudioRequest(BaseModel):
     input_annotation: S3Path
     output_bucket: str
     validation_type: ValidationType
-    deadline: Optional[datetime]
+    deadline: Optional[datetime] = None
```
Comment (spiridonovfed) [inline convert/src/label_studio_to_badgerdoc/models/label_studio_models.py:23]: There's no point of adding "= None" as soon as "Optional[datetime] does the same.
It also applies to all other chenges in this commit

**Review by spiridonovfed — CHANGES_REQUESTED** [review]

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py (hunk 1 above)
Comment (serereg) [reply convert/src/label_studio_to_badgerdoc/models/label_studio_models.py]: Fixed

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py (hunk 3 above)
Comment (serereg) [reply convert/src/label_studio_to_badgerdoc/models/label_studio_models.py]: Fixed

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py (hunk 5 above)
Comment (serereg) [reply convert/src/label_studio_to_badgerdoc/models/label_studio_models.py:23]: pyright shows warning without " = None"

File: convert/src/label_studio_to_badgerdoc/models/label_studio_models.py (hunk 2 above)
Comment (serereg) [reply convert/src/label_studio_to_badgerdoc/models/label_studio_models.py]: Fixed

**Pull request closed without merging**